src/test/java/org/elasticsearch/test/InternalTestCluster.java

```diff
@@ -277,9 +277,10 @@ public InternalTestCluster(long clusterSeed, Path baseDir,
         logger.info("Setup InternalTestCluster [{}] with seed [{}] using [{}] data nodes and [{}] client nodes", clusterName, SeedUtils.formatSeed(clusterSeed), numSharedDataNodes, numSharedClientNodes);
         this.settingsSource = settingsSource;
         Builder builder = ImmutableSettings.settingsBuilder();
-        if (random.nextInt(5) == 0) { // sometimes set this
+        // nocommit - currently we force multi data dir
+        if (random.nextInt(5) == 0 || true) { // sometimes set this
             // randomize (multi/single) data path, special case for 0, don't set it at all...
-            final int numOfDataPaths = random.nextInt(5);
+            final int numOfDataPaths = 1 + random.nextInt(5);
```

Comment:
I guess the 1+ is bevause of the no commit? but we are already under an ` if (random.nextInt(5) == 0)`. I think `if (numOfDataPaths > 0)` is not needed?